jerry-core/lit/lit-char-helpers.c

```diff
@@ -44,14 +124,10 @@ bool
 lit_char_is_space_separator (ecma_char_t c) /**< code unit */
 {
   /* Zs */
-#define LIT_UNICODE_RANGE_ZS(range_begin, range_end) \
-  if (c >= (range_begin) && c <= (range_end)) \
-  { \
-    return true; \
-  }
-#include "lit-unicode-ranges.inc.h"
-
-  return false;
+  return (c == LIT_CHAR_SP || c == LIT_CHAR_NBSP // most common ones
+         || (c >= unicode_separator_char_interv_sps[0]
+            && c <= unicode_separator_char_interv_sps[0] + unicode_separator_char_interv_lens[0])
+         || search_char_in_char_array (c, unicode_separator_chars, NUM_OF_ELEMENTS (unicode_separator_chars)));
 } /* lit_char_is_space_separator */
```

Comment:
Please optimize this for ASCII:
if (c <= 127)
{
  return c == ... || c == .. etc
}
else
{
 return ...;
}
or optimizing lit_char_is_white_space is even better